### ISSUE TITLE
Dynamic Links not working when using nativescript-fabric on iOS

### ISSUE THREAD

**Issue by surdu** [EddyVerbruggen/nativescript-plugin-firebase#778]
When using this plugin and [nativescript-fabric](https://github.com/hypery2k/nativescript-fabric) at the same time on iOS, the `onDynamicLinkCallback` is not called anymore.

I managed to trace the line that's causing issue to [this one in nativescript-fabric](https://github.com/hypery2k/nativescript-fabric/blob/89f308c286d5031b2f875ce05c663539c7d22919/src/fabric.ios.ts#L20).

I guess it could be written in a more non-intrusive manner to only set the `applicationDidFinishLaunchingWithOptions` on the appDelegate if the delegate is already set, but I'm guessing nativescript-plugin-firebase also needs this method.

Any idea how this kind of conflict can be solved ?

**Comment by EddyVerbruggen** [comment]
Yeah, plugins fighting over appdelegate is a common problem. This plugin tries to play nice with other plugins, but others boldly override all of it. 

Perhaps in you case you can use Crashlytics which I recently integrated in this plugin.
